test.sh

```diff
@@ -20,8 +21,7 @@ yarn lint
 # Run unit tests
 yarn test
 
-# Run end-to-end tests
-# TODO
-
-# Upload code coverage
-./prow-codecov.sh 2>/dev/null
+# Upload code coverage (CI environment only)
+if [[ -n "${CI:=}" ]]; then
+  ./prow-codecov.sh 2>/dev/null
```

Comment:
@vojtechszocs I realized after we talked that this script only runs for non-local jobs. See the check here: https://github.com/openshift/dynamic-plugin-sdk/blob/main/prow-codecov.sh#L14

So this new check is redundant...